carp.common/src/commonMain/kotlin/dk/cachet/carp/common/ddd/ApplicationServiceEventBus.kt

```diff
@@ -20,22 +20,22 @@ class ApplicationServiceEventBus<
     /**
      * Subscribe to events of [eventType] belonging to [applicationServiceKlass] and handle them using [handler].
      */
-    suspend fun <
+    fun <
```

Comment:
I tried making a bit more legible by using shorter generic names. But, the main reason for adding the distinct generic types on separate lines is because they use nested generics.

I could shorten the generic types even futher, but overall, even if that allows me to place it all on one line, I personally don't find `<OS : ApplicationService<OS, OSE>, OSE: IntegrationEvent<OS>>` more legible.